e2etest/pkg/frr/container.go

```diff
@@ -23,16 +23,18 @@ const (
 	frrMountPath = "/etc/frr"
 )
 
+var containerRuntime = executor.GetContainerRuntime()
```

Comment:
Let's have a `Runtime` global variable here, sounds better.
We init it in an init() function and we refer it in the executor.

Reply:
maybe better keep it in the executor file? as we already import the executor to this file

Reply:
Ah yes, in that case it won't work because go does not allow circular dependencies